### ISSUE TITLE
Popovers with library details have unusable links

### ISSUE THREAD

**Issue by stof** [StateOfJS/StateOfJS#142]
On hover on the name of a library, a popover appears with extra informations about it, including links to its website. But these links are unusable. As soon as the mouse moves (to reach this link), the popover disappears.

**Comment by SachaG** [comment]
Yes, that happens due to how some browsers (Firefox at least) handle SVG hovers. I think the only good solution would be taking the tooltips out of the SVG but then we'd need to have a way to still position them correctly above the labels…

See also: https://stackoverflow.com/questions/53274575/extending-an-svg-tooltips-hover-zone-out-of-its-parent-elements-bounds
